apps/monograph/app/components/monographpost/index.tsx

```diff
@@ -90,6 +90,13 @@ function generateTableOfContents() {
         : level;
     currentHeading = headingLevel;
 
+    const isCalloutHeading =
+      !heading.previousSibling &&
```

Comment:
Why this check?

Reply:
If I don't use this check, the user defined headings inside callout aren't added to the toc

Reply:
We don't want any headings inside a callout to appear in TOC.

Reply:
ah, got it. I misunderstood then.